documentation.go

```diff
@@ -498,7 +498,11 @@ func (d *documentationCommand) formatFlags(c Command, info *Info) string {
 			if i > 0 {
 				theFlags += ", "
 			}
-			theFlags += fmt.Sprintf("`--%s`", f.Name)
+			if len(f.Name) == 1 {
+				theFlags += fmt.Sprintf("`-%s`", f.Name)
```

Comment:
I hate this variable name (although I know it wasn't your idea). Better to name it `flagsString` or `formattedFlags` or something.

Reply:
Agreed